products/amazon-cdk.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Amazon CDK
+category: framework
+iconSlug: amazonaws
+permalink: /amazon-cdk
+changelogTemplate: https://github.com/aws/aws-cdk/releases/tag/v__LATEST__
+releaseDateColumn: true
+
+auto:
+-   git: https://github.com/aws/aws-cdk
+
+identifiers:
+-   repology: aws-cdk
+-   purl: pkg:npm/aws-cdk
```

Comment:
Add a few more PURLs for the cdk's go repo:

```suggestion
-   purl: pkg:npm/aws-cdk
-   purl: pkg:golang/github.com/aws/aws-cdk-go
-   purl: pkg:github/aws/aws-cdk-go
```

Reply:
Amazing wasn't aware of it